VAMobile/src/components/CommonErrorComponents/ErrorComponent.tsx

```diff
@@ -29,13 +33,53 @@ const ErrorComponent: FC<ErrorComponentProps> = (props) => {
   const features = ScreenIDToDowntimeFeatures[props.screenID]
   const isInDowntime = oneOfFeaturesInDowntime(features, downtimeWindowsByFeature)
 
-  const getSpecificErrorComponent: FC<ErrorComponentProps> = ({ onTryAgain, screenID }) => {
+  const getSpecificErrorComponent: FC<ErrorComponentProps> = ({ onTryAgain, screenID, error }) => {
     const tryAgain = onTryAgain ? onTryAgain : storeTryAgain
     const errorType = errorsByScreenID[screenID] || ''
 
     if (isInDowntime) {
       return <DowntimeError screenID={screenID} />
     }
+
+    if (error && isErrorObject(error)) {
+      const reactQueryErrorType = getCommonErrorFromAPIError(error, screenID)
+      switch (reactQueryErrorType) {
```

Comment:
```suggestion
      switch (errorType) {
```

Reply:
when we migrate the error slice and can fully drop line 38
const errorType = errorsByScreenID[screenID] || ‘’
then we can make that change. currently it gives a linting error by having it as errorType